features/mesh/Index.feature

```diff
@@ -2,7 +2,7 @@ Feature: mesh / index
   Background:
     Given the CSS selectors
       | Alias          | Selector                                     |
-      | items          | [data-testid='mesh-collection']              |
```

Comment:
Please could you leave this as collection

Reply:
We already had *-table selectors in two other places so I stuck with that to not further have different patterns for this.

Reply:
Everything has moved to use `AppCollection` please could you use the term 'collection' for multiple things. You've actually changed them back to what we don't want here. Please could you use `collection` terminology, or at the very least leave where we are using `collection` terminology as it is.

Reply:
> to what we don't want here.

You. I was perfectly happy with calling those tables tables.

I changed those things now.